pkg/api/conn_track.go

```diff
@@ -19,6 +19,11 @@ package api
 
 import "time"
 
+const (
+	HashIdFieldName     = "NetObs-HashId"
```

Comment:
Please remove "NetObs"

Reply:
I talked to @eranra offline and he is not fond of the mention of "NetObs" in the prefix.
@mariomac What about using underscore "_" as the prefix? Similar practice is employed in elasticsearch.
```go
const (
	HashIdFieldName     = "_HashId"
	RecordTypeFieldName = "_RecordType"
)
```